src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/id/BaseIdTest.java

```diff
@@ -0,0 +1,56 @@
+package fi.otavanopisto.kuntaapi.test.server.unit.id;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertTrue;
+
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+import org.reflections.Reflections;
+
+import fi.otavanopisto.kuntaapi.server.id.BaseId;
+
+public class BaseIdTest {
+
+  @Test
+  public void testTestUniqueHashSeeds() {
+    Reflections reflections = new Reflections("fi.otavanopisto.kuntaapi.server");
+    
+    List<BaseId> ids = new ArrayList<>();
+    
+    for (Class<? extends BaseId> idClass : reflections.getSubTypesOf(BaseId.class)) {
+      try {
+        if (!Modifier.isAbstract(idClass.getModifiers())) {
+          ids.add(idClass.newInstance());
+        }
+      } catch (InstantiationException | IllegalAccessException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)